src/platform/datapath_raw.h

```diff
@@ -17,7 +17,9 @@ typedef struct CXPLAT_SOCKET_POOL {
 
 } CXPLAT_SOCKET_POOL;
 
-typedef struct CXPLAT_DATAPATH CXPLAT_DATAPATH;
+#define MANGLE(x) XDP_##x
```

Comment:
What is this for?

Reply:
just mimicking C++ mangling to make function name same, but avoid build error and search easily. It will be deleted by refactoring